[BUG] Build fails against cudf 0.11

Attempting to build cuSpatial with cuDF branch-0.11 fails with the following:
```
-- Build files have been written to: /rapids/cuspatial/cpp/build
Scanning dependencies of target cuspatial
[  6%] Building CUDA object CMakeFiles/cuspatial.dir/src/io/shp/polygon_shapefile_reader.cu.o
[ 12%] Building CUDA object CMakeFiles/cuspatial.dir/src/io/soa/polygon_soa_reader.cu.o
[ 18%] Building CUDA object CMakeFiles/cuspatial.dir/src/io/soa/point_soa_reader.cu.o
[ 25%] Building CUDA object CMakeFiles/cuspatial.dir/src/io/soa/uint32_soa_reader.cu.o
[ 31%] Building CUDA object CMakeFiles/cuspatial.dir/src/io/soa/timestamp_soa_reader.cu.o
[ 37%] Building CUDA object CMakeFiles/cuspatial.dir/src/query/spatial_window_points.cu.o
/rapids/cuspatial/cpp/src/query/spatial_window_points.cu:18:10: fatal error: utilities/cuda_utils.hpp: No such file or directory
 #include <utilities/cuda_utils.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
CMakeFiles/cuspatial.dir/build.make:127: recipe for target 'CMakeFiles/cuspatial.dir/src/query/spatial_window_points.cu.o' failed
make[2]: *** [CMakeFiles/cuspatial.dir/src/query/spatial_window_points.cu.o] Error 1
CMakeFiles/Makefile2:75: recipe for target 'CMakeFiles/cuspatial.dir/all' failed
make[1]: *** [CMakeFiles/cuspatial.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
```

It looks like [the recent move of utilities to the legacy subfolder](https://github.com/rapidsai/cudf/pull/3422) is causing it.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,7 @@
 
 - PR #94 Add legacy headers as cudf migrates
 - PR #98 Updates to accommodate cudf refactoring
+- PR #103 Update the include paths for cuda_utils
 
 
 # cuSpatial 0.10.0 (16 Oct 2019)

--- a/cpp/src/query/spatial_window_points.cu
+++ b/cpp/src/query/spatial_window_points.cu
@@ -15,7 +15,7 @@
  */
 
 #include <cudf/utilities/legacy/type_dispatcher.hpp>
-#include <utilities/cuda_utils.hpp>
+#include <utilities/legacy/cuda_utils.hpp>
 #include <type_traits>
 #include <thrust/device_vector.h>
 #include <thrust/count.h>

--- a/cpp/src/spatial/hausdorff.cu
+++ b/cpp/src/spatial/hausdorff.cu
@@ -15,7 +15,7 @@
  */
 
 #include <cudf/utilities/legacy/type_dispatcher.hpp>
-#include <utilities/cuda_utils.hpp>
+#include <utilities/legacy/cuda_utils.hpp>
 #include <type_traits>
 #include <thrust/device_vector.h>
 

--- a/cpp/src/spatial/haversine.cu
+++ b/cpp/src/spatial/haversine.cu
@@ -20,7 +20,7 @@
 #include <cudf/utilities/legacy/type_dispatcher.hpp>
 #include <cudf/legacy/column.hpp>
 
-#include <utilities/cuda_utils.hpp>
+#include <utilities/legacy/cuda_utils.hpp>
 #include <type_traits>
 #include <thrust/device_vector.h>
 #include <utility/utility.hpp>

--- a/cpp/src/spatial/lonlat_to_coordinate.cu
+++ b/cpp/src/spatial/lonlat_to_coordinate.cu
@@ -15,7 +15,7 @@
  */
 
 #include <cudf/utilities/legacy/type_dispatcher.hpp>
-#include <utilities/cuda_utils.hpp>
+#include <utilities/legacy/cuda_utils.hpp>
 #include <type_traits>
 #include <utility>
 #include <thrust/device_vector.h>

--- a/cpp/src/spatial/point_in_polygon.cu
+++ b/cpp/src/spatial/point_in_polygon.cu
@@ -15,7 +15,7 @@
  */
 
 #include <cudf/utilities/legacy/type_dispatcher.hpp>
-#include <utilities/cuda_utils.hpp>
+#include <utilities/legacy/cuda_utils.hpp>
 #include <type_traits>
 #include <utility/utility.hpp>
 #include <cuspatial/point_in_polygon.hpp>

--- a/cpp/src/trajectory/derive_trajectories.cu
+++ b/cpp/src/trajectory/derive_trajectories.cu
@@ -17,7 +17,7 @@
 #include <type_traits>
 
 #include <cudf/utilities/legacy/type_dispatcher.hpp>
-#include <utilities/cuda_utils.hpp>
+#include <utilities/legacy/cuda_utils.hpp>
 #include <rmm/thrust_rmm_allocator.h>
 
 #include <utility/utility.hpp>

--- a/cpp/src/trajectory/subset_trajectories.cu
+++ b/cpp/src/trajectory/subset_trajectories.cu
@@ -16,7 +16,7 @@
 
 #include <cudf/utilities/legacy/type_dispatcher.hpp>
 #include <cudf/legacy/copying.hpp>
-#include <utilities/cuda_utils.hpp>
+#include <utilities/legacy/cuda_utils.hpp>
 #include <type_traits>
 #include <thrust/device_vector.h>
 #include <thrust/binary_search.h>

--- a/cpp/src/trajectory/trajectory_distance_speed.cu
+++ b/cpp/src/trajectory/trajectory_distance_speed.cu
@@ -15,7 +15,7 @@
  */
 
 #include <cudf/utilities/legacy/type_dispatcher.hpp>
-#include <utilities/cuda_utils.hpp>
+#include <utilities/legacy/cuda_utils.hpp>
 #include <type_traits>
 #include <thrust/device_vector.h>
 

--- a/cpp/src/trajectory/trajectory_spatial_bound.cu
+++ b/cpp/src/trajectory/trajectory_spatial_bound.cu
@@ -15,7 +15,7 @@
  */
 
 #include <cudf/utilities/legacy/type_dispatcher.hpp>
-#include <utilities/cuda_utils.hpp>
+#include <utilities/legacy/cuda_utils.hpp>
 #include <type_traits>
 
 #include <utility/utility.hpp>

